icons/gauge-circle.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "../icon.schema.json",
+  "tags": [
+    "dashboard",
+    "dial",
+    "meter",
+    "speed",
+    "pressure",
+    "measure",
+    "level"
+  ],
+  "categories": [
+    "account",
+    "transportation",
+    "sports",
```

Comment:
why is this part of sports?

Reply:
Oh I see was already part of `gauge` check.

Reply:
> why is this part of sports?

> Oh I see was already part of `gauge` check.

@ericfennis Ah yeah, and in answer to the question, specifically, motorsports…